### DISCUSSION TITLE
Update browser support

### DISCUSSION THREAD

**Discussion by o-t-w** [jquery/jquery#4988] [Q&A] [answered]
I understand that solving browser support issues has always been one of the main reasons jQuery has been popular. However, we are currently at a time where [most open source projects have stopped supporting Internet Explorer 11](https://dev.to/hypeddev/internet-explorer-finally-dies-26cn). e.g. Vue 3, Preact 11, Angular 13, Ember 4, Ionic, Marko, AMP from Google, SolidJS, Bootstrap, Tailwind, etc

Have you considered dropping support for Internet Explorer 9 and 10? That would allow you to modernise the codebase somewhat and reduce the size of the library.

**Comment by mgol** [comment, marked as answer]
We've already dropped IE<11 support in #4347 and Edge Legacy in #4792. We consider reducing our browser support matrix a breaking change, though, so this will only get released in jQuery 4.0.0 and we still have a lot of items to do for that release. In the meantime, we'll continue releasing 3.x updates with current browser support.
